src/helpers/math.js

```diff
@@ -0,0 +1,153 @@
+import {isArray} from '../util/utils.js';
+
+/**
+ * Determine whether or not two values are equal (===).
+ * Example usage:
+ *      {{eq '3' 3}}    => false
+ *
+ * @param value1
+ * @param value2
+ * @returns boolean
+ */
+function eq(value1, value2) {
```

Comment:
These are not `math` helpers they are rather logical, conditional or comparison helpers. I don't think they should be in math.js